### ISSUE TITLE
Invalid hyperlinks (spaces are included but shouldn't)

### ISSUE THREAD

**Issue by EgorBo** [ForNeVeR/kaiwa#110]
Steps to repoduce:
1) Paste this to the chat:
https://github.com/WebAssembly/ilwasm  CIL t
2) the whole text including "  CIL" will be highlighted as a hypelink

**Comment by ForNeVeR** [comment]
Note that it seems to be critical that there are two spaces before "CIL".

**Comment by unam3** [comment]
Root of all evil is src/js/helpers/htmlify.js
parseLinks function cannot distinguish escaped html in url:

"http://tost.in  cc cc" → "<a href="http://tost.in&nbsp;&nbsp;cc" target="_blank">http://tost.in&nbsp;&nbsp;cc</a> cc"

How fix this? Update linkify?

Eh…Check http://tost.in&nbsp;&nbsp;cc in http://formvalidation.io/validators/uri/ — is github url parsing broken too?

**Comment by ForNeVeR** [comment]
Does updated Linkify solves the problem?

**Comment by unam3** [comment]
correct markdown shit:
```
"http://tost.in  cc cc" → "<a href="http://tost.in&nbsp;&nbsp;cc" target="_blank">http://tost.in&nbsp;&nbsp;cc</a> cc"
```

**Comment by ForNeVeR** [comment]
Was fixed by #111, thanks to @unam3.
